procedures/katello/release.md.erb

```diff
@@ -20,7 +20,7 @@
   - [ ] For any cherry-picks that are not needed (including Redmine trackers) you can add them to the `:ignores:` section of `tool_belt` in `configs/katello/<%= short_version %>.yaml`
 <% end -%>
 <% unless is_rc -%>
-- [ ] Change Redmine version <%= full_version %> state to Closed using <%= rel_eng_script('close_redmine_version') %>
+- [ ] Change Redmine version <%= full_version %> state to Closed using <%= rel_eng_script('close_redmine_version') %>: `REDMINE_API_KEY=<secret> PROJECT=katello VERSION=<%= short_version %> ./close_redmine_version`
```

Comment:
It prompts for `REDMINE_API_KEY` and I'm not sure we should encourage this. The key does end up in your bash history. We can also encourage storing it in gopass, like we do with other secrets.

As for the env vars: that's really what we need for all `rel_eng_script`. I see there's already precedent for it in this release procedure. I tend to use `export PROJECT=... VERSION=...` at the start.

Any thoughts?